client/src/pages/reports/Show.js

```diff
@@ -661,6 +670,32 @@ const ReportShow = ({ setSearchQuery, pageDispatchers }) => {
                     />
                   }
                 />
+
+                <Field
+                  name="attachments"
+                  label={Settings.fields.attachment.shortLabel}
+                  component={FieldHelper.ReadonlyField}
+                  humanValue={
+                    <div
+                      id="attachmentCardList"
+                      style={{
+                        display:
+                          report.attachments.length > 1
+                            ? "-webkit-box"
```

Comment:
`display: -webkit-box` is non-standard and AFAICT appears to be unsupported in any browser.